.circleci/config.yml

```diff
@@ -64,7 +64,7 @@ commands:
 jobs:
   runtest:
     macos:
-      xcode: "12.0"
+      xcode: "12.2.0"
```

Comment:
both `pod lib lint` and the carthage integration step were failing on 12.0. I'm really not sure what was going on, though - 12.0 does compile locally and `deployment_checks` pass locally on 12.0 on my computer. 
It would not find `SKPaymentQueue`'s `presentCodeRedemptionSheet`, which is weird because it was added on 12.0. 
Really not sure what was going on there, but this should at least fix the tests for the time being. 
If we want to be compatible with Xcode 11, though, we could add a compiler flag similar to the one in [ErrorUtils](https://github.com/RevenueCat/purchases-ios/blob/develop/Purchases/Public/RCPurchasesErrorUtils.m#L199).